scripts/maven/Dependencies.kt

```diff
@@ -0,0 +1,10 @@
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/** Data class that stores the list of dependencies present in maven_install.json. */
+@JsonClass(generateAdapter = true)
+data class Dependencies(
+
+  @Json(name = "dependencies") val dependencyList: MutableList<Dependency>
```

Comment:
For this & other files: why MutableList? Why not just List?

Reply:
Because once they are generated, we also need to sort them. And I think we can't sort a simple List.

Reply:
In that case we should be copying the data when we need to change it. Models generally should be immutable--that offers a bunch of benefits in data management & reduction in complexity.